style.css

```diff
@@ -3,7 +3,6 @@ html, body {
     padding: 0;
     font-family: sans-serif;
 }
-
```

Comment:
Don't remove white lines between declarations. They are good for readability. Can you add them back?